src/napari_manual_transforms/_model.py

```diff
@@ -165,6 +179,26 @@ def compact_axis_angle(self):
     def compact_axis_angle(self, ca):
         self.quaternion = rot.quaternion_from_compact_axis_angle(ca)
 
+    @property
+    def translation(self) -> np.ndarray:
+        return self._t
+
+    @translation.setter
+    def translation(self, t: Sequence[float] | np.ndarray):
+        """x,y,z"""
+        self._t = np.asarray(t)
```

Comment:
it looks like a strict assumption is made in `self.transform` that `self.translation` has three items correct?  If so, then we need some sort of validation to be done here in the setter.

Reply:
Yes, that is correct. I now convert the input to an array and check its dimension/size afterwards.
If it does not have a length of 3, a ValueError will be raised.

Also removed the type hint for allowing np.ndarrays here as quaternion inputs are similar to translation inputs but also only expect inputs of type `Sequence`.